bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -184,7 +184,10 @@ <h2>{{ ftl('home-mozilla-makes-privacy') }}</h2>
     ) %}
     <h2>{{ ftl('home-is-mozilla-a-corporation')}}</h2>
     <p>{{ ftl('home-mozilla-consists-of') }}</p>
-    <a href="{{ url('mozorg.about.manifesto') }}" class="mzp-c-button">{{ ftl('home-learn-about-mofo') }}</a>
+    <a href="https://foundation.mozilla.org/" class="mzp-c-button" rel="external noopener">
```

Comment:
Do we want to add some `data-*` attributes so this will be logged in GA?

Reply:
i don't quite remember why, but i recall them only wanting data attributes on the product CTAs!